axlearn/cloud/gcp/tpu.py

```diff
@@ -718,7 +718,7 @@ def infer_tpu_workers(tpu_type: str) -> int:
             tpu_version, tpu_cores = match.groups()
             if tpu_version in {"v3", "v4", "v5p"}:
                 return int(tpu_cores) // 8
-            if tpu_version in {"v5litepod"}:
+            if tpu_version in {"v5litepod", "v6e"}:
```

Comment:
Can you rebase main? v6e is already supported

Reply:
Done